packages/ag-charts-community/src/options/series/cartesian/bulletOptions.ts

```diff
@@ -16,7 +17,17 @@ export interface AgBulletSeriesTooltipRendererParams<TDatum = any>
     extends AgSeriesTooltipRendererParams<TDatum>,
         BulletSeriesKeysAndNames {}
 
-export interface AgBulletSeriesThemeableOptions extends AgBaseSeriesThemeableOptions {}
+export interface AgBulletSeriesStyle extends AgBarSeriesStyle {}
+
+export interface AgBulletScaleOptions {
+    /** Maximum value of the scale. Any values exceeding this number will be clipped to this maximum */
+    max?: number;
```

Comment:
The way to achieve this with other chart-types is via the axis options - we should follow the same approach for this.

See https://ag-grid.com/javascript-charts/axes-domain/#domain-min--max

Reply:
I thought of that, but @AG-DavidG and I discussed this. We came to the conclusion that bullet graphs are gauges that don't conceptually have any axes (even though the underlying implementation axes). So having a dedicated `scale.max` makes sense in this case.

Although we could perhaps just have `scale.max` be an alias for `axes[0].domain.max` to make it easier to maintain.